### ISSUE TITLE
IP Change Broke extension

### ISSUE THREAD

**Issue by RobertLowe** [codysherman/2NFM#46]
Heads up,

Cheers

**Comment by codysherman** [comment]
Uhhhhh wow. Didn't know anyone else is using it lol. We just switched off of the original demo server to our own custom server we're hosting. Haha, we'll have to come up with a solution here, I now realize I have no idea how many people are using it, I hope this free server holds up lmao.

**Comment by codysherman** [comment]
To answer your question, if you pull the latest changes, it should now work. We had a half step where we put in a custom IP before setting up SSL, it's now live live.

**Comment by RobertLowe** [comment]
No worries, I pulled a copy changed the IP and ran the websocket server for myself 👍

**Comment by RobertLowe** [comment]
Yeah, it 's just a signalling server it should be very low traffic, something you could run on heroku easily for free, IMO

**Comment by codysherman** [comment]
Awesome, yeah we're having issues where sometimes the client just doesn't show anything playing, so we decided to fork the server to see if we can figure out the issue. Seems especially likely when on the same network as the streaming extension. Let me know if you have any ideas.

**Comment by RobertLowe** [comment]
Sounds like an issue at the SDP level, or perhaps TURN/STUN servers.

**Comment by codysherman** [comment]
Hi @RobertLowe just following up here in case you run into the same issue. Yes it is the TURN server, I'm very new to this whole WebRTC world, and I had to learn what a TURN server as I debugged the app haha. But you were right. The original repo I forked from had a TURN server built in which was recently turned off, as I assume the extension grew popular enough to cost him too much money.

I put in credentials for a dev account from xirsys.com and confirmed it solved my issue. Here is the current ticket I'm working out of that I'll keep in the loop: https://github.com/codysherman/2NFM/issues/51#issuecomment-533909586
